### ISSUE TITLE
Release Planner Install Error

### ISSUE THREAD

**Issue by BusinessNone** [microsoft/powerapps-tools#717]
Release Planner has a CE Dependency that isn't included in the Solutions. 

There are missing dependencies. Install the following solutions before installing this one: "msdynce_ProductManagement (9.0.4.0066)"

**Comment by beigewell** [comment]
I get this same error

**Comment by ljrain** [comment]
Release Planner uses the Product from the Product Catalog.

**Comment by BusinessNone** [comment]
> Release Planner uses the Product from the Product Catalog.

Why? It isn't like Product is a restricted entity. And so wouldn't this really then be a Dynamics 365 App instead of a Power App? Doesn't tying this to Dynamics 365 kind of go against the design principles?

**Comment by yodelmas** [comment]
Yep same dependencies "There are missing dependencies. Install the following solutions before installing this one: "msdynce_ProductManagement (9.0.4.0066)"

=> So a D365 Sales App is needed in order to use it. +1 @BusinessNone

**Comment by BusinessNone** [comment]
> Yep same dependencies "There are missing dependencies. Install the following solutions before installing this one: "msdynce_ProductManagement (9.0.4.0066)"
> 
> => So a D365 Sales App is needed in order to use it. +1 @BusinessNone

Which makes this a Dynamics 365 tool and not a Power App tool.
